maintainers/maintainer-list.nix

```diff
@@ -1,6 +1,6 @@
 /* List of NixOS maintainers.
-    ```nix
```

Comment:
Please undo all the formatting changes

Reply:
I removed them and fixed commit messages.